Automattic/wp-calypso
# Migrate GMB styles to webpack

#### Changes proposed in this Pull Request

* Migrate _Google My Business_ styles to webpack

#### Testing instructions

* Check the _Google My Business_ tab in stats (on a Business site)
* Make sure to check the tab with and without connected GMB

related #27515
